topics/data-management/tutorials/0013_file-naming/slides.html

```diff
@@ -0,0 +1,187 @@
+
+![special characters to use](../../images/0013/02.png)
+
+???
+
+- A filename can be both human and machine readable and as a general rule the use of special characters is avoided.  
+
+- So, what is a special character?  
+
+- A special character is any character that isn’t a number or a letter.  
+
+- Although we avoid the use of special characters in filenames, we will explain later that the use of the full stop, the underscore, and the minus sign are commonplace and useful.
+
+---
+
+## Characters in Filenames to Avoid: Special Characters
+
+.left[The forward slash (/) denotes a directory]
+
+![cat command](../../images/0013/03.png)
+
+???
+
+- Special characters are often used by programming languages to denote a special property or function.
+
+- One example you may know is the forward slash which denotes a directory.  A filename containing a forward slash character causes problems.  
+
+- In this example the Unix cat command is used to read the contents of a file.   Here the command throws an error because the filename contains a forward slash.  The command instead tries to look for a file called post qc dot txt under a directory called pre. 
+
+## Characters to Avoid: Spaces
+
+.left[Space characters are interpreted as spaces between different files]
+
+![cat command](../../images/0013/04.png)
+
+???
+
+- A space character can also cause problems in a filename even though it is used frequently
+
+- A file with a single space character in its filename can sometimes be interpreted by a script as two separate files. 
+
+- In this example there is a filename with three space characters.  Where we try to read the contents of this using the cat command in Unix, we get an error because the cat command thinks there are four separate files.
+
+---
+
+## Characters to Avoid: Spaces
+
+.left[Using characters to separate words in a filename]
+
+![cat command](../../images/0013/05.png)
+
+???
+
+---
+
+## Characters to Avoid: Spaces
+
+.left[Using dot (.) underscore (_) and dash (-) in different contexts]
+
+![cat command](../../images/0013/06.png)
+
+???
+
+---
+
+## Machine-readable Filenames
+
+![cat command](../../images/0013/07.png)
+
+???
+
+---
+
+## Machine-readable Filenames
+
+![cat command](../../images/0013/08.png)
+
+???
+
+---
+
+## Using Numbers at the Beginning of Filenames
+
+.left[Padding with zeros]
+
+![cat command](../../images/0013/09.png)
+
+???
+
+---
+
+## Using Dates at the Beginning of Filenames
+
+.left[Placing dates at the beginning of the filename]
+
+![cat command](../../images/0013/10.png)
+
+.left[ISO standard]
+
+![cat command](../../images/0013/11.png)
+
+???
+
+---
+
+## Human-readable Filenames
+
+.left[Bad]
+
+![cat command](../../images/0013/12.png)
+
+.left[Better]
+
+![cat command](../../images/0013/13.png)
+
+???
+
+---
+
+## File Versioning
+
+.left[Bad]
+
+![cat command](../../images/0013/14.png)
+
+.left[Better]
+
+![cat command](../../images/0013/15.png)
+
+???
```

Comment:
It seems you're missing speaker notes for a few of these slides. The videos will not be able to compile without something to say per slide.